NDB.Covid19/NDB.Covid19.Droid/Views/AuthenticationFlow/InformationAndConsentActivity.cs

```diff
@@ -125,19 +125,6 @@ void GoToErrorPage(AuthErrorType error)
             });
         }
 
-        //After calling this method you cannot return by going "Back".
-        //OnCreate has to be called again if returning to this page.
-        void GoToQuestionnairePage()
```

Comment:
you have removed the method but not the calls of the method. it will not build

Reply:
Fixed